src/plugins/auto_batch/src/sync_infer_request.cpp

```diff
@@ -52,7 +52,7 @@ void SyncInferRequest::share_tensors_with_batched_req(const std::set<std::string
         auto batched_tensor = m_batched_request_wrapper->_infer_request_batched->get_tensor(it);
         if (!batched_tensor._so)
             batched_tensor._so = m_batched_request_wrapper->_infer_request_batched._so;
-        res = create_shared_tensor_on_batched_tensor(batched_tensor, name, batched_inputs, m_batch_id, m_batch_size);
+        res = create_shared_tensor_on_batched_tensor(batched_tensor, std::move(name), batched_inputs, m_batch_id, m_batch_size);
```

Comment:
Name is std::string, why it trigger scan issue?

Reply:
string has move constructor and can use here, we are trying to remove potential copy here